src/otx/algorithms/anomaly/tasks/openvino.py

```diff
@@ -172,15 +168,13 @@ def infer(self, dataset: DatasetEntity, inference_parameters: InferenceParameter
         if inference_parameters is not None:
             update_progress_callback = inference_parameters.update_progress  # type: ignore
 
-        # This always assumes that threshold is available in the task environment's model
-        meta_data = self.get_metadata()
         for idx, dataset_item in enumerate(dataset):
-            image_result = self.inferencer.predict(dataset_item.numpy, metadata=meta_data)
+            image_result: AnomalyResult = self.inference_model(dataset_item.numpy)
 
             # TODO: inferencer should return predicted label and mask
-            pred_label = image_result.pred_score >= 0.5
-            pred_mask = (image_result.anomaly_map >= 0.5).astype(np.uint8)
-            probability = image_result.pred_score if pred_label else 1 - image_result.pred_score
+            pred_label = image_result.pred_label
+            pred_mask = image_result.pred_mask
+            probability = image_result.pred_score if pred_label == "Anomaly" else 1 - image_result.pred_score
```

Comment:
I think in other parts of the code base we use `anomalous` as the label name, not `Anomaly`, so maybe it would be good to use the same here for consistency. Though I guess this would involve making changes on model api side.